packages/@tinacms/cli/src/next/commands/init-command/index.ts

```diff
@@ -4,8 +4,9 @@ import { initStaticTina } from '../../../cmds/init'
 
 export class InitCommand extends Command {
   static paths = [['init']]
-  rootPath = Option.String('--rootPath', {
-    description: 'Specify the root directory to run the CLI from',
+  rootPath = Option.String('--forestryPath', {
```

Comment:
We should might want to consider calling the variable `forestryPath` since we use `rootPath` in other places and it has a different meaning.

Reply:
Updated!